tests/conftest.py

```diff
@@ -817,36 +817,52 @@ def get_markets():
             'symbol': 'ETH/USDT',
             'base': 'ETH',
             'quote': 'USDT',
+            'settle': 'USDT',
+            'baseId': 'ETH',
+            'quoteId': 'USDT',
+            'settleId': 'USDT',
+            'type': 'spot',
             'spot': True,
-            'future': True,
-            'swap': True,
             'margin': True,
-            'type': 'spot',
-            'contractSize': None,
+            'swap': True,
+            'future': True,
+            'option': False,
+            'active': True,
+            'contract': True,
+            'linear': True,
+            'inverse': False,
+            'taker': 0.0006,
+            'maker': 0.0002,
+            'contractSize': 1,
+            'expiry': 1680220800000,
+            'expiryDateTime': '2023-03-31T00:00:00.000Z',
```

Comment:
Is this really the desired pair to apply all of these futures values to?

i don't think an expiry makes sense on a spot market ... (and in my understanding, futures markets would be `ETH/USDT:USDT`)

Reply:
I removed a bunch of them, swap and future were already set to True, and if I change that, other tests fail and the problem spiders

Reply:
it's not per se a problem - i just noticed "quite some" of them being added in this PR ... which made me wonder :)